Doc/c-api/intro.rst

```diff
@@ -174,6 +174,39 @@ complete listing.
    .. versionchanged:: 3.8
       MSVC support was added.
 
+.. c:macro:: PyDoc_STRVAR(name, str)
+
+   Creates a variable with name ``name`` that can be used in docstrings.
+   If Python is built without docstrings, the value will be empty.
+
+   Use :c:macro:`PyDoc_STRVAR` for docstrings to support building
+   Python without docstrings, as specified in :pep:`7`.
+
+   Example::
+
+      PyDoc_STRVAR(pop_doc, "Remove and return the rightmost element.");
+
+      static PyMethodDef deque_methods[] = {
+          // ...
+          {"pop", (PyCFunction)deque_pop, METH_NOARGS, pop_doc},
+          // ...
+      }
+
+.. c:macro:: PyDoc_STR(str)
+
+   Creates a docstring for the given input string or an empty string
+   if docstrings are disabled.
+
+   Use :c:macro:`PyDoc_STR` in specifying docstrings to support
+   building Python without docstrings, as specified in :pep:`7`.
+
+   Example::
+
+      static PyMethodDef pysqlite_row_methods[] = {
```

Comment:
Can you modify this example so that it doesn't reference any third party modules (`pysqlite` in this case)? We typically avoid doing that in the documentation unless it's specifically relevant, especially for code examples.

Reply:
@aeros can you clarify what you mean by 3rd-party modules?  This example is taken from CPython at https://github.com/python/cpython/blob/f669581a9527afb0d2325f9845a86715c0ba365d/Modules/_sqlite/row.c#L223.  Do you mean that we should not include any actual examples from CPython?  In that case we would want to change the `deque_methods` in this PR also.

Reply:
> can you clarify what you mean by 3rd-party modules? This example is taken from CPython

Never mind, you can disregard the above comment. There's a 3rd party package on PyPI titled `pysqlite` which I thought this was in reference to, I hadn't realized this was a function in a standard library module (which is perfectly okay to use as a reference in the docs). Thanks for clearing that up.